specs/phase0/validator.md

```diff
@@ -289,8 +289,14 @@ def is_candidate_block(block: Eth1Block, period_start: uint64) -> bool:
 def get_eth1_vote(state: BeaconState, eth1_chain: Sequence[Eth1Block]) -> Eth1Data:
     period_start = voting_period_start_time(state)
     # `eth1_chain` abstractly represents all blocks in the eth1 chain sorted by ascending block height
-    votes_to_consider = [get_eth1_data(block) for block in eth1_chain if
-                         is_candidate_block(block, period_start)]
+    votes_to_consider = [
+        get_eth1_data(block) for block in eth1_chain
+        if (
+            is_candidate_block(block, period_start)
+            # Ensure only forward progress
+            and get_eth1_data(block).deposit_count > state.eth1data.deposit_count
```

Comment:
If there are no new deposits, we can't finalize the eth1-blockhash part anymore, stalling progress more than helping it. Should this be a `>=`?